internal/mount/flag.go

```diff
@@ -96,3 +116,26 @@ func MetadataCacheTTL(statCacheTTL, typeCacheTTL time.Duration, ttlInSeconds int
 
 	return
 }
+
+// StatCacheMaxSizeMB returns the stat-cache size in MiBs based on the user old and new flags/configs.
+func StatCacheMaxSizeMB(mountConfigStatCacheMaxSizeMB int64, flagStatCacheCapacity int) (statCacheMaxSizeMB uint64, err error) {
+	if mountConfigStatCacheMaxSizeMB != config.StatCacheMaxSizeMBUnsetSentinel {
+		if mountConfigStatCacheMaxSizeMB == -1 {
+			statCacheMaxSizeMB = config.MaxSupportedStatCacheMaxSizeMB
+		} else {
+			statCacheMaxSizeMB = uint64(mountConfigStatCacheMaxSizeMB)
+		}
+	} else {
+		if flagStatCacheCapacity != DefaultStatCacheCapacity {
+			if flagStatCacheCapacity < 0 {
+				return 0, fmt.Errorf("invalid value of stat-cache-capacity (%v), can't be less than 0", flagStatCacheCapacity)
+			}
+			avgTotalStatCacheEntrySize := AverageSizeOfPositiveStatCacheEntry + AverageSizeOfNegativeStatCacheEntry
```

Comment:
in gcsfuse v1, if user specifies stat-cache-capacity it doesn't translate to positive + negative. So now we are abstracting out the details of positive + negative from user and hence we should update the flag's help also to say that the average size is 1320 (1200 for positive and 120 for negative).

Reply:
done.